source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -70,11 +69,7 @@ public async Task Request()
             var serverSupportsChannels = await ServerSupportsChannels();
             commandOutputProvider.Debug(serverSupportsChannels ? "This Octopus Server supports channels" : "This Octopus Server does not support channels");
 
-            commandOutputProvider.Debug("Finding project: {Project:l}", ProjectName);
```

Comment:
It is in the extension method, more or less - I just moved it to after the fetch so we could log name and id together, because I noticed the logs full of ids looked too mysterious. We discussed it on slack.